src/nvim/version.c

```diff
@@ -378,11 +378,11 @@ static const int included_patches[] = {
   // 1546,
   1545,
   // 1544,
-  // 1543,
+  1543,
   // 1542,
   1541,
   // 1540,
-  // 1539,
+  1539,
```

Comment:
the changes to verison.c should be reverted. This version.c still refers to Vim 8.0, not 8.1. And the changes aren't needed anyway, because the `vim-patch:xx` tokens in the commit messages will be noticed by our scripts/CI/etc.